### PR TITLE
 [Close #37] - Solved 1º Challenge

### LINKED ISSUE #37: Solve challenge 1

### PR DESCRIPTION
**Pull request by SoBatista**

### CODE REVIEW

File: challenge_30_days/day_1.py [hunk 1]
```diff
@@ -1,8 +1,23 @@
 '''
     Write a Python program that asks the user for a test grade from 0 to 100 and prints the the grade in letter form:
     A - 100-85
```
Comment (SoBatista) [inline challenge_30_days/day_1.py:3]: You need to change to:
  A - 85-100
  B -  66-84
  C -  50-65
  D -  25-49
  E -   0-24
You have the first line incorrect: A - 100-85
And the second as well: B - 65-84

Comment (MRegra) [reply challenge_30_days/day_1.py:3]: Ok

File: challenge_30_days/day_1.py [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
     D -  25-49
     E -   0-24
-'''
\ No newline at end of file
+'''
+
+grade = int(input("What is your grade from 0 to 100? "))
+
+if grade in range(0, 25):
+    print("This is your letter form: E")
```
Comment (MRegra) [inline challenge_30_days/day_1.py:13]: Can you do: Your grade is E? 
Same for the others

File: challenge_30_days/day_1.py [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+elif grade in range(25, 50):
+    print("This is your letter form: D")
+elif grade in range(50, 66):
+    print("This is your letter form: C")
+elif grade in range(66, 85):
+    print("This is your letter form: B")
+elif grade in range(85, 101):
+    print("This is your letter form: A")
+else:
+    print("That does not exist!")
```
Comment (MRegra) [inline challenge_30_days/day_1.py:23]: Can you throw an error instead?

File: challenge_30_days/day_1.py [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
     Write a Python program that asks the user for a test grade from 0 to 100 and prints the the grade in letter form:
     A - 100-85
-    B -  65-84
+    B -  66-84
     C -  50-65
     D -  25-49
     E -   0-24
-'''
\ No newline at end of file
+'''
```
Comment (MRegra) [inline challenge_30_days/day_1.py:8]: Can you create a function?

**Pull request closed without merging**